src/main/java/core/basesyntax/impl/Element.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.impl;
+
+public class Element<K, V> {
```

Comment:
```suggestion
public class Node<K, V> {
```
usually classes that store elements in collections are called Node
also lets make this class an inner class in StorageImpl - because is should not be used outside that class